packages/l1connection/l1connection.go

```diff
@@ -30,8 +35,12 @@ type Config struct {
 type Client interface {
 	// requests funds from faucet, waits for confirmation
 	RequestFunds(addr iotago.Address, timeout ...time.Duration) error
-	// sends a tx, waits for confirmation
-	PostTx(tx *iotago.Transaction, timeout ...time.Duration) error
+	// sends a block (including tipselection and local PoW if necessary)
+	PostBlock(block *iotago.Block, timeout ...time.Duration) (*iotago.Block, error)
+	// sends a tx (including tipselection and local PoW if necessary)
+	PostTx(tx *iotago.Transaction, timeout ...time.Duration) (*iotago.Block, error)
+	// sends a tx (including tipselection and local PoW if necessary) and waits for confirmation
+	PostTxAndWaitUntilConfirmation(tx *iotago.Transaction, timeout ...time.Duration) (*iotago.Block, error)
```

Comment:
```suggestion
	PostTxAndWaitUntilConfirmation(tx *iotago.Transaction, timeout ...time.Duration) (*iotago.Block, error)
```
seems only `PostTxAndWaitUntilConfirmation` is used, no need to expose the other funcions

Reply:
Oh sorry, automerge was enabled :sweat_smile: 
Should I remove these in the next PR?

Reply:
yeah, just noticed after approving. sure :+1: